packages/terra-status-view/src/terra-dev-site/doc/status-view/DprecationGuide.2.doc.mdx

```diff
@@ -0,0 +1,12 @@
+## Deprecation Notice
```

Comment:
File name spell correction should be 'DeprecationGuide'

Reply:
updated here https://github.com/cerner/terra-core/pull/3363/commits/763eb983f6c0078d75d40715852d7ffaeb38e422.